magento/magento2
# Performance | Remove Redundant Review Rating Info loading

### Description (*)
This PR fixes the problem that is fully described here - https://github.com/magento/magento2/issues/37938

### Fixed Issues (if relevant)
1. Fixes  https://github.com/magento/magento2/issues/37938

### Manual testing scenarios (*)
1. Add Review with Rating vote for Product
2. Go to admin and accept that review
3. Go to product and check that Review is shown with correct votes


### Contribution checklist (*)
